app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -125,60 +153,63 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
         )
     }
 
+    private fun onSyncNow() {
```

Comment:
Rename this to `syncNow` - again, aiming for consistency with the interactor parameters.